src/libnvme.map

```diff
@@ -1,7 +1,5 @@
 LIBNVME_1_0 {
 	global:
-		__nvme_get_log_page;
```

Comment:
You can't remove this one. This is a public function (even documented) see https://github.com/linux-nvme/libnvme/blob/5aea0216cfb2410f9cdb7e2e25d04fbb4022a317/src/nvme/linux.h#L83

Bad name. We should rename it. Though I don't have a better name yet.